File: solutions/beecrowd/1134/1134.cpp
#include <cstdint>
#include <cstdio>

int main() {
    std::int32_t n, v[] = {0, 0, 0};

    while (scanf("%d", &n) && n != 4) {
        if (n >= 1 && n <= 3) {
            v[n - 1]++;
        }
    }

    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", v[0]);
    printf("Gasolina: %d\n", v[1]);
    printf("Diesel: %d\n", v[2]);

    return 0;
}

File: solutions/beecrowd/1134/1134.py
counts = [0, 0, 0]
n = int(input())
while n != 4:
    if n in (1, 2, 3):
        counts[n - 1] += 1
    n = int(input())

print('MUITO OBRIGADO')
print(f'Alcool: {counts[0]}')
print(f'Gasolina: {counts[1]}')
print(f'Diesel: {counts[2]}')
